### PR TITLE
Clarify build config in VS/IIS dev-time support topic

### LINKED ISSUE #9531: How to run a release build?
Following this approach IIS references the project root directory containing web.config which launches the process "bin\IISSupport\VSIISExeLauncher.exe". But when you switch to a Release build IISExeLauncherArgs.txt still contains a path to the debug build.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f5477f20-bdbf-ae0b-63b4-8bf6789f7a48
* Version Independent ID: fa9dc362-76a4-5d11-f457-b3d8550f9997
* Content: [Development-time IIS support in Visual Studio for ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/development-time-iis-support?view=aspnetcore-2.1)
* Content Source: [aspnetcore/host-and-deploy/iis/development-time-iis-support.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/host-and-deploy/iis/development-time-iis-support.md)
* Product: **aspnet-core**
* GitHub Login: @shirhatti
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #9531 

[Internal Review Topic (Run the project section)](https://review.docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/development-time-iis-support?view=aspnetcore-1.0&branch=pr-en-us-9541#run-the-project)

Thanks @Neutrino-Sunset for the issue.

### CODE REVIEW

**Comment by shirhatti** [comment]
Sounds like a VS issue. I'd wait for Bill to respond on the original issue before changing the docs here

**Comment by guardrex** [comment]
Will do, but I guessed from the result of trying it that it is by-design ...

capture

**Comment by guardrex** [comment]
ping @BillHiebert

**Comment by guardrex** [comment]
@shirhatti The commit is in ....

Follows up from the email with a single line that gives the reader a heads-up on the warning that VS will throw with a Release config and dev-time support.

**Comment by guardrex** [comment]
@scottaddie Bill said offline that the remark ...

> Debugging a Release build configuration with [Just My Code](/visualstudio/debugger/just-my-code) and compiler optimizations results in a degraded experience (for example, break points aren't hit).

... is ok. That gives a dev a heads-up on the warning that they'll get from VS if they try to use dev-time support with a Release build config.

The other change here is trivial .......... if I got the commas in the right spots. 😝lol

File: aspnetcore/host-and-deploy/iis/development-time-iis-support.md [hunk 1]
```diff
@@ -123,14 +123,20 @@ Alternatively, manually add a launch profile to the [launchSettings.json](http:/
 
 ## Run the project
 
-In the VS UI, set the Run button to the **IIS** profile and select the button to start the app:
+In the VS UI:
```
Comment (scottaddie) [inline aspnetcore/host-and-deploy/iis/development-time-iis-support.md]:
```suggestion
In Visual Studio:
```

File: aspnetcore/host-and-deploy/iis/development-time-iis-support.md [hunk 2]
```diff
@@ -123,14 +123,20 @@ Alternatively, manually add a launch profile to the [launchSettings.json](http:/
...
+* Set the Run button to the **IIS** profile and select the button to start the app.
+
+![The Run button in the VS toolbar is set to the IIS profile with the build configuration drop-down list set to Release.](development-time-iis-support/_static/toolbar.png)
 
 Visual Studio may prompt a restart if not running as an administrator. If prompted, restart Visual Studio.
 
 If an untrusted development certificate is used, the browser may require you to create an exception for the untrusted certificate.
 
+> [!NOTE]
+> Debugging a Release build configuration with [Just My Code](/visualstudio/debugger/just-my-code) and compiler optimizations results in a degraded experience (for example, break points aren't hit).
```
Comment (scottaddie) [inline aspnetcore/host-and-deploy/iis/development-time-iis-support.md]:
```suggestion
> Debugging a Release build configuration with [Just My Code](/visualstudio/debugger/just-my-code) and compiler optimizations results in a degraded experience. For example, break points aren't hit.
```

**Comment by guardrex** [comment]
@shirhatti I'm going to go ahead with this based on Bill's approval. If you would like me to make changes to it, hit me up ... you know how to find me. 😀

**Pull request merged**